### ISSUE TITLE
[visual-recognition] "Error - unspecified error - please try again" 

### ISSUE THREAD

**Issue by holly-cummins** [watson-developer-cloud/node-sdk#699]
We've been able to reproduce this, with about 80% likelihood, on versions 3.30 - 3.5.2 of the node-sdk. I'm not sure if it's a client issue or a backend issue, but we didn't see it with version 2.5.x (instead, we saw #698). 

When processing a zip of images, we get an error  "Error - unspecified error - please try again". If we retry, we get "socket hang up" and an ECONNRESET.

**Comment by dpopp07** [comment]
Hi @holly-cummins is this still an issue in current versions of the SDK / service?

**Comment by holly-cummins** [comment]
We were able to work around the issue by reducing the number of images in our zips from 20 to 4. In the end we decided zips of 4 were barely worth it and we also did a lot of single image uploads. I think the underlying issue might have been a lack of parallelism in the service when processing batched zips, which caused the response times to be long enough that the http connection would drop. 
Unfortunately, I'm not able to check if it's still an issue, but if it is an issue it would be more on the service side, so we can close this.
